index.js

```diff
@@ -0,0 +1,45 @@
+import { addBook } from './modules/addBook.js';
+import { showListBook } from './modules/showListBook.js';
+import { deleteBook } from './modules/deleteBook.js';
+import { DateTime } from './modules/luxon.js';
+
+const displayTime = document.getElementById('date');
+const datetime = DateTime.utc().toLocaleString(DateTime.DATETIME_FULL);
```

Comment:
- [ ] Great job using the Luxon library to display the current date below the navigation bar for this requirement and also import from its module directory. :100: